### DISCUSSION TITLE
[Plugin Docs] Include basic setup example for plugins

### DISCUSSION THREAD

**Discussion by Andre601** [woodpecker-ci/woodpecker#2835] [Request a Feature]
It should be considered to have a basic setup included in the plugin pages, if specific information is available.

Said info could be provided through the YAML frontmatter of the docs.md file. That way can we easily find instructions on how it would be added to woodpecker and how the settings may look like in the end...

**Comment by qwerty287** [comment]
Do yyou mean the plugin-specific docs?
These are generated from files made by the plugin authors, so they can add detailed examples.

**Comment by qwerty287** [comment]
I'm going to close this. Please contact plugin authors if you want to have documentation on specific plugins.
